src/components/Table/Table.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
```

Comment:
I suspect we can't make this a PureComponent as the nested data object might change, but we would want the table to still rerender.

Reply:
Ah yeah - it just does the shallow compare. I changed it because eslint was [barking about needing to be a stateless function](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md) because I pulled out the uneeded constructor.

I'll refactor into a stateless function and update the PR